src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -177,12 +187,16 @@ internal float ParentOriginY
             get
             {
                 float temp = 0.0f;
```

Comment:
How about change "temp" as "temporary" or "returnValue" ?

https://readlearncode.com/code-and-stuff/clean-code-variables/
: "Use words that can be pronounced and avoid abbreviations"

Reply:
Done